main.go

```diff
@@ -159,6 +161,8 @@ func defineFlagVar() *FlagVar {
 		"Indicates the failure max delay in seconds")
 	flag.StringVar(&flagVar.finalState, "final-state", string(v1alpha1.StateReady),
 		"Customize final state, to mimic state behaviour like Ready, Warning")
+	flag.StringVar(&flagVar.finalDeletionState, "final-deletion-state", string(v1alpha1.StateDeleting),
+		"Customize final state when module marked for deletion, to mimic state behaviour like Ready, Warning")
```

Comment:
```suggestion
	flag.StringVar(&flagVar.finalDeletionState, "final-deletion-state", "",
		"Customize final state when module marked for deletion, to mimic state behaviour like Ready, Warning")
```

Default should be empty! Please also look at the other comments